### DISCUSSION TITLE
E73-2G4M08S1(E or C) for Zigbee?

### DISCUSSION THREAD

**Discussion by se7entynine** [rbaron/b-parasite#210] [Q&A] [answered]
Hey, 
I have just a short question because the datasheets of each module seemed to not mention it. 

My plan is to build the zigbee version of the b-parasite and I wanted to order 20 pcs. through JLCPCB. 
The datasheets of the [E73-2G4M08S1E](https://www.cdebyte.com/products/E73-2G4M08S1E/4#Downloads) chip clearly mention the Zigbee functionality, but the [E73-2G4M08S1C](https://www.cdebyte.com/products/E73-2G4M08S1C/4#Downloads) ones doesn't. 
[The comparision site of Nordic states](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_nrf52%2Fstruct%2Fnrf52.html) that they can utilize Zigbee but I just wanted to make sure that the ones of JLCPCB will work for the zigbee version. What statement is true?

Additionally they have multiple versions for the B3U-1000P - [is this a suitable one](https://jlcpcb.com/partdetail/OmronElectronics-B3U1000P/C231329)?

Thanks in advance and also for this great project!

**Comment by rbaron** [comment, marked as answer]
Hey @se7entynine,

Yes, the [E73-2G4M08S1C](https://www.ebyte.com/en/downpdf.aspx?id=445) is a good choice for all the samples in the repo. It's the one I use, but the [E73-2G4M08S1E](https://www.cdebyte.com/products/E73-2G4M08S1E/4#Downloads) should also work just fine.

> Additionally they have multiple versions for the B3U-1000P - [is this a suitable one](https://jlcpcb.com/partdetail/OmronElectronics-B3U1000P/C231329)?

It should work well from a quick look at the offering.

Happy hacking!
